packages/app/src/vis-packs/core/line/MappedLineVis.tsx

```diff
@@ -85,7 +91,7 @@ function MappedLineVis(props: Props) {
   }, [disableErrors, errors]);
 
   useEffect(() => {
-    // Disable `autoScale` for 1D datasets (baseArray and dataArray are the same)
+    // Disable `autoScale` for 1D datasets (baseArray and dataArray are the same) or if the value is a slice
```

Comment:
Are you sure about this? `dims` is the shape of the full dataset, not the slice. I think it still works the same, which is good.

Reply:
Sorry, leftover from my previous attempt, I will remove this